pkgs/tools/networking/aria2/default.nix

```diff
@@ -13,10 +12,10 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [ pkgconfig ];
-  buildInputs = [ openssl c-ares libxml2 sqlite zlib libssh2 ] ++
-    stdenv.lib.optional stdenv.isDarwin Security;
+  buildInputs = [ openssl c-ares libxml2 sqlite zlib libssh2 ];
 
-  configureFlags = [ "--with-ca-bundle=/etc/ssl/certs/ca-certificates.crt" ];
+  configureFlags = [ "--with-ca-bundle=${cacert}/etc/ssl/certs/ca-bundle.crt" ] ++
+    stdenv.lib.optional stdenv.isDarwin [ "--with-openssl" "--without-appletls" ];
```

Comment:
Tried:

* `--with-apple-tls` (without ca bundle)
* `--with-apple-tls --with-ca-bundle`
* flags from makerelease-osx.mk 
* flags from makerelease-osx.mk and `--with-ca-bundle`

None worked. This is the result:

```
Exception: [AbstractCommand.cc:345] errorCode=1 URI=https://www.example.com/
  -> [SocketCore.cc:1015] errorCode=1 SSL/TLS handshake failure: Invalid certificate format
fish: “result/bin/aria2c https://www.e…” terminated by signal SIGSEGV (Address boundary error)
```

I tried tracing back from `SocketCore.cc`, and found that this is caused by Security.framework returning [errSSLBadCert](https://developer.apple.com/library/mac/documentation/Security/Reference/secureTransportRef/index.html#//apple_ref/c/econst/errSSLBadCert). I haven't looked into libsecurity_ssl itself, but at very least Nix's `curl` on Darwin seems to also built with OpenSSL rather than SecureTransport (AppleTLS).